include/browseview_func.php

```diff
@@ -733,7 +727,7 @@ function getVernacularEntries($letter = "", $langcode = "", $page)
 	if(count($arrEntries) == 0)
 	{
 
-		$sql = "SELECT SQL_CALC_FOUND_ROWS ID, post_title, post_content, search_strings ";
+		$sql = "SELECT SQL_CALC_FOUND_ROWS $wpdb->posts.ID, post_title, post_content, search_strings ";
```

Comment:
A better solution would be to alias the `$wp->posts` table. It would be less confusing.

Reply:
Yes, indeed :-)